packages/core/lib/src/ui-models/menu_list_item_data.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/cupertino.dart';
+
+/// Data model for a menu item
+///
+/// e.g. Sidebar, Drawer, ..etc.
+class MenuListItemData {
```

Comment:
I'm not sure if this is something you are okay with or find as a good practice. 
I usually do this where I create models to help me create UI data like menu lists. What do you think? @jenshor

Reply:
I usually do this as well, however, I'm not always sure if this is the best practice since it also makes mocking the Widget a bit harder. In the end it's your choice.